VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -562,3 +575,12 @@ def get_prev_log(self):
             return ''.join(str(x) for x in lines)
         else:
             return ''.join(str(x) for x in lines)
+    
+    def get_shell_script_log(self):
+        with open(self._context._shell_log_file, "r") as f:
+            lines = f.readlines()
+        if(len(lines) > 10):
+            lines = lines[-10:]
+            return ''.join(str(x) for x in lines)
```

Comment:
move return after if-else, no need of else

Reply:
done